### DISCUSSION TITLE
笔记保存到DaliyNotes引用问题反馈

### DISCUSSION THREAD

**Discussion by mal5338** [zhaohongxuan/obsidian-weread-plugin#175] [General]
将笔记保存到DaliyNotes时，引用的ID与实践的ID不一致，多了一个图书的ID，导致无法正常展示。
<img width="550" alt="image" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/5944407/ce77ea18-22e5-4746-b4bf-e58665c9776c">
<img width="1057" alt="image" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/5944407/7dd53b7f-0a53-43d0-a7bf-2a527cda385c">
<img width="1056" alt="image" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/5944407/51f3ec94-4835-4a44-a1cf-f06485a62626">

**Comment by mal5338** [comment]
去掉书名ID后可以正常展示，希望修复。
<img width="523" alt="image" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/5944407/625f6138-8e59-404a-9bf2-41bdc1621a1c">
<img width="1071" alt="image" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/5944407/287d59b9-7462-4d59-9082-534a60a840b0">

**Comment by weqoocu** [comment]
遇到同样的问题

**Comment by looktooz** [comment]
如果想修改这个dailynote中的id，要在哪里修改？
